ArduPlane/ArduPlane.cpp

```diff
@@ -310,6 +310,12 @@ void Plane::one_second_loop()
     // indicates that the sensor or subsystem is present but not
     // functioning correctly
     gcs().update_sensor_status_flags();
+
+#if HAL_WITH_IO_MCU
```

Comment:
It would be nice to be able to put this into IOMCU directly - otherwise we need this patch in the other vehicles.  There's no 1Hz loop but you could run a hal-millis-based timer in something which will always be called.

This would involve passing in a bitmask to the iomcu library at init.

Reply:
Yeah, I really debated it, but at the implementation time I wasn't sold on if calling logger from other threads was safe. If it is, then it fixes the race condition I outlined above. If it's not then you could stall the IO comm link on accounting of logging, which seems like a bad thing (tm)